content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -187,24 +187,26 @@ example, to run version {{< param "fullversion" >}}, you would run the following
 minikube start --kubernetes-version {{< param "fullversion" >}}
 ```
 #### Specifying the VM driver
-You can change the VM driver by adding the `--vm-driver=<enter_driver_name>` flag to `minikube start`.
+You can change the VM driver by adding the `--driver=<enter_driver_name>` flag to `minikube start`.
 For example the command would be.
 ```shell
-minikube start --vm-driver=<driver_name>
+minikube start --driver=<driver_name>
 ```
  Minikube supports the following drivers:
  {{< note >}}
- See [DRIVERS](https://git.k8s.io/minikube/docs/drivers.md) for details on supported drivers and how to install
+ See [DRIVERS](https://minikube.sigs.k8s.io/docs/reference/drivers/) for details on supported drivers and how to install
```

Comment:
```suggestion
See [Drivers](https://minikube.sigs.k8s.io/docs/reference/drivers/) for details on supported drivers and how to install
```

Reply:
@sftim please see my recent comment, hopefully this assists on the technical review aspect.